devel/juno-1/bootstrap.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+CLEANUP=${CLEANUP:-"0"}
+
+GREEN='\e[0;32m'
+RESET='\e[0m'
+RED='\e[0;31m'
+
+function log {
+    echo -e "${GREEN} /-------------------------------------------------------------------------------------- ${RESET}"
+    echo -e "${GREEN} - $@ ${RESET}\n"
+}
+
+function die {
+    echo -e "${RED} /-------------------------------------------------------------------------------------- ${RESET}"
+    echo -e "${RED} - $@ ${RESET}\n"
+    exit 1
+}
+
+PASSWORD=${PASSWORD:-1234567890}
+CHAIN_ID=${CHAIN_ID:-juno-1}
+MONIKER_NAME="$(hostname)"
+JUNO_HOME="juno_home"
+JUNO_GENESIS_HEIGHT="2578099"
+CHAIN_REPO="https://raw.githubusercontent.com/CosmosContracts/mainnet/main/$CHAIN_ID" && \
+export PEERS="$(curl -s "$CHAIN_REPO/persistent_peers.txt")"
+
+if [[ $CLEANUP -eq "1" ]]; then
+  log "Deleting all local data"
+  rm -rf ./tmp/ > /dev/null
+fi
+
+
+log "Setting up working directory"
+mkdir -p tmp
+pushd tmp
+
+log "Copy junod to tmp"
+if [ $(which junod) ]; then
+  cp $(which junod) ./
+else
+  die "junod not found in PATH. Please install junod before running this script."
+fi
+
+log "Initialize the chain"
+if [ ! -d ${JUNO_HOME} ]; then
+  junod init "$MONIKER_NAME" --chain-id $CHAIN_ID --home ${JUNO_HOME}
+fi
+
+log "Download the genesis file"
+if [ ! -f ${JUNO_HOME}/config/.genesis.downloaded ]; then
+  curl -s https://share.blockpane.com/juno/phoenix/genesis.json > ${JUNO_HOME}/config/genesis.json
+  touch ${JUNO_HOME}/config/.genesis.downloaded
+fi
+
+log "Set persistent peers"
+sed -i -e "s/^persistent_peers *=.*/persistent_peers = \"$PEERS\"/" ${JUNO_HOME}/config/config.toml
+
+log "Set Halt Height"
+sed -i -e "s/^halt-height = 0$/halt-height = 2616300/" ${JUNO_HOME}/config/app.toml
```

Comment:
Should this halt height be configurable?

Reply:
I'm planning to also write an upgrade.sh script. I'll do that when we have binary as for gaia-dm.